docs/security/host-certs.md

```diff
@@ -87,14 +87,55 @@ If your institution is not in the list and Let's Encrypt certificates do not mee
 
 1. Install the [host](#installing-host-certificates) or [service](#installing-service-certificates) certificate
 
-Requesting Let's Encrypt Certificates
--------------------------------------
+Requesting Host Certificates Using [Let's Encrypt](https://letsencrypt.org/)
+----------------------------------------------------------------------------
+
+[Let's Encrypt](https://letsencrypt.org/) is a free, automated, and open CA frequently used for web services;
+see the [security team's position on Let's Encrypt](https://opensciencegrid.github.io/security/LetsEncryptOSGCAbundle/)
+for more details.  Let's Encrypt can be used to obtain host certificates as an alternative
+to InCommon if you do not have an InCommon subscription.
```

Comment:
Let's put this sentence on a new line (https://opensciencegrid.org/technology/documentation/style-guide/#meta) and instead of "you", let's use "your institution"

Reply:
Roger.